src/metabase/lib/drill_thru/automatic_insights.cljc

```diff
@@ -21,6 +21,7 @@
    stage-number                                 :- :int
    {:keys [column column-ref dimensions value]} :- ::lib.schema.drill-thru/context]
   (when (and (lib.drill-thru.common/mbql-stage? query stage-number)
+             (not= (:lib/source column) :source/native)
```

Comment:
FIXME! This was done to disable this drill for native queries

Reply:
I'm not actually sure I can think of a better way to disallow this. We could mark freshly-wrapped queries somehow, but that marker wouldn't get saved with the question.